### PR TITLE
chore: setup connect-go proto files

### PR DESCRIPTION
**Pull request by AndreasZeissner**
## Motivation and Context

This is used for https://github.com/wundergraph/terraform-provider-cosmo.

The package can be used like this: 

```
go get github.com/wundergraph/cosmo@638540986b469dxxxxxxxxxxxxxxxxxx
```

## TODO

- [ ] Tests or benchmark included
- [ ] Documentation is changed or added on [https://app.gitbook.com/](https://app.gitbook.com/)
- [ ] PR title must follow [conventional-commit-standard](https://github.com/wundergraph/wundergraph/blob/main/CONTRIBUTING.md#conventional-commit-standard)

### CODE REVIEW

**Review by StarpTech — CHANGES_REQUESTED** [review]
How do we detected outdated artifacts? Is it running when the controlplane changes?

**Comment by AndreasZeissner** [comment]
Our pipelines will fail when the git directory is dirty. This happens on router builds at least automatically.

Didn't saw this as an artifact, clients using it have to ensure those are up to date which is at the moment the terraform-provider which needs to adapt to whatever changes here.

**Comment by StarpTech** [comment]
Does it mean a workflow will fail when the code becomes outdated in this repo?

**Comment by AndreasZeissner** [comment]
Not necessarily in the terraform provider. It will fail there when e2e tests are failing which are running against latest cosmo. 

I think it's a good idea to keep the files updated in general.
Nevertheless I think no one will do unless you need to, or the pipeline fails. This would be  latest in the router builds as they are running the generate-go command and do the dirty check, even though it's unrelated.

The controlpalne ci seems to be the only place where this would make sense, even when publishing it somewhere later.

Are there any considerations on reusing things on the router?

File: buf.connect-go.go.gen.yaml [hunk 1]
```diff
@@ -0,0 +1,16 @@
+version: v2
+managed:
+  enabled: true
+  disable:
+    - file_option: go_package
+      module: buf.build/googleapis/googleapis
+  override:
+    - file_option: go_package_prefix
+      value: github.com/wundergraph/cosmo/connect-go
```
Comment (StarpTech) [inline buf.connect-go.go.gen.yaml:9]: Let's stick with the convention of "gen". We do this for router and collector.

**Review by StarpTech — APPROVED** [review]
LGTM

**Pull request merged**